### ISSUE TITLE
Chromium is not rendering anything at MacOS High Sierra

### ISSUE THREAD

**Issue by schelkun** [puppeteer/puppeteer#2435]
Just installed latest puppeteer version (1.3.0), Chromium r549031 was downloaded.
But I can't see anything inside headful browser.

![2018-04-24 15 41 45](https://user-images.githubusercontent.com/31924884/39187625-793ab28c-47d6-11e8-9050-bac28718b319.png)

**Comment by schelkun** [comment]
When I'm trying to launch Chromium from puppeteer from Finder it works fine.
But when Chromium was launched by node (pptr) or terminal I have this bug and some messages in console:

```
[2058:775:0424/173958.523461:ERROR:render_widget_host_view_base.cc(159)] Not implemented reached in virtual uint32_t content::RenderWidgetHostViewBase::GetCaptureSequenceNumber() const
```

**Comment by aslushnikov** [comment]
@schelkun from the screenshot, looks like webfont didn't get loaded.

I run pptr on High Sierra without any problems; this looks like an issue with your environment. I'd try playing with defaultArgs to figure what flag causes the weird behavior. Another option would be asking over on chromium-dev google group.

Hope this helps.

**Comment by StephenGrider** [comment]
@aslushnikov Not sure its an env issue, I helped out someone today with the same issue.  We fixed it by downgrading to v1.0.0 of puppeteer, which includes an earlier release of Chromium.
